channel/funder.go

```diff
@@ -186,6 +188,21 @@ func (f *Funder) Fund(ctx context.Context, request channel.FundingReq) error {
 	return nonFundingErrg.Err()
 }
 
+func (f *Funder) fundingTimeoutContext(ctx context.Context, req channel.FundingReq) (context.Context, context.CancelFunc, error) {
+	timeout, err := NewBlockTimeoutDuration(ctx, f.ContractInterface, req.Params.ChallengeDuration)
+	if err != nil {
+		return nil, nil, errors.WithMessage(err, "creating block timeout")
+	}
+	ctx, cancel := context.WithCancel(ctx)
+	go func() {
+		if err := timeout.Wait(ctx); err != nil && !pcontext.IsContextError(err) {
+			f.log.Warn("Fund: BlockTimeout.Wait runtime error: ", err)
+		}
+		cancel() // cancel funding context on funding timeout
```

Comment:
```suggestion
		cancel() // Cancel funding context on funding timeout.
```

Reply:
The code was just moved. The intention is not to cleanup existing code here. Just to satisfy the linter with regards to function length.